frontend/src/scenes/insights/insightSceneLogic.tsx

```diff
@@ -244,14 +277,27 @@ export const insightSceneLogic = kea<insightSceneLogicType>([
                 return
             }
 
+            const dashboardName = dashboardLogic.findMounted({ id: dashboard })?.values.dashboard?.name
```

Comment:
the `findMounted` calls need to be here, get some redux/kea issues while calling them in the `breadcrumbs:` selector as that selector because of how that selector is called in `breadcrumbsLogic`